src/Apps/Artwork/Components/ArtworkImageBrowser/SaveUtilButton.tsx

```diff
@@ -15,12 +15,17 @@ export const SaveUtilButton: FC<SaveUtilButtonProps> = ({
   return (
     <UtilButton
       name="heart"
-      Icon={isSaved ? FilledIcon : HeartStrokeIcon}
+      Icon={isSaved ? UnsaveIcon : SaveIcon}
       label={isSaved ? "Saved" : "Save"}
       longestLabel="Saved"
       onClick={onClick}
     />
   )
 }
 
-const FilledIcon = () => <HeartFillIcon title="Unsave icon" fill="blue100" />
+const SaveIcon = () => (
+  <HeartStrokeIcon data-testid="save-icon" title="Save icon" fill="blue100" />
```

Comment:
**question**: I think there is no need/we should not pass `fill="blue100"` here

Reply:
Good catch! 

46d325f3a62bc6cb48c0dd2b6ab00378133844cf